packages/playground/src/components/account/account-eth-form/account-eth-form.tsx

```diff
@@ -11,6 +11,7 @@ export class AccountEthForm {
   @Prop() header: string = "";
   @Prop() button: string = "";
   @Prop() disabled: boolean = false;
+  @Prop() min: number = 0.01;
```

Comment:
Is this all that's required to restrict the min amount? I seem to be able to deposit less than this still... :/

Reply:
Nope, it isn't enough. Forgot something.

Reply:
This change only covered `/deposit`, not `/exchange`.

Reply:
Fixed!